src/vivarium_gates_child_iv_iron/data/loader.py

```diff
@@ -86,7 +108,8 @@ def filter_population(unfiltered: pd.DataFrame) -> pd.DataFrame:
 
 
 def load_age_bins(key: str, location: str) -> pd.DataFrame:
-    return interface.get_age_bins()
+    all_age_bins = interface.get_age_bins().reset_index()
```

Comment:
This should have updated this in the artifact code but I didn't catch it until running observers.